scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -350,6 +352,7 @@ task PopulateFilterSetInfo {
 
     String query_project
 
+    File monitoring_script = "gs://gvs_quickstart_storage/cromwell_monitoring_script.sh"
```

Comment:
was the goal to leave the monitoring script in here?

Reply:
One of the issues I ran into with the intermediate callset was this step running out of space but, without any monitoring, I had to guess what to change it to.  Because the next run will be even more data, I wanted to leave this here in case it happened again.